spec/lib/defra_ruby_companies_house_spec.rb

```diff
@@ -70,4 +70,14 @@
       end
     end
   end
+
+  context "when the service is down" do
+    before do
+      allow_any_instance_of(DefraRubyCompaniesHouse).to receive(:load_company).and_raise(StandardError)
```

Comment:
I think this should be raising one of the RestClient exception types - the test is to confirm that it get mapped to StandardError

Reply:
Actually there should be one test each for (a) when the company number is not valid (ResourceNotFound) and (b) when the service is down (NotFound)